src/services/search/__tests__/listingSearch.test.ts

```diff
@@ -70,6 +74,61 @@ describe("SEARCH service", () => {
     })
   })
 
+  describe("#fetchDealerListingsCount", () => {
+    const dealerId = 123
+    const requestOptionsMock = {
+      accessToken: "DUMMY TOKEN",
+    }
+
+    it("unwraps count from json", async () => {
+      const count = 400
+      fetchMock.mockResponse(JSON.stringify({ count }))
+
+      const response = await fetchDealerListingsCount({
+        dealerId,
+        options: requestOptionsMock,
+      })
+      expect(response).toEqual(count)
```

Comment:
Does this test make sense like that, I might miss something but isn't it just ensuring that fetch has been called in any way at some point in the stack...
Maybe it would make more sense to mock `fetchPath` in the base file and ensure that it was called with the correct props instead of testing if the fetch mock returns the mocked data...

Reply:
Saw that the same may apply to other tests following the same pattern, no gonna comment there so we can have the discussion in one place

Reply:
I mostly moved those around from the old implementation. It does ensure that the API response is processed the way we expect. We mainly mock raw fetch in the package that implements a lot of function on top of fetch to have a better overview if something breaks in the implementation of the wrappers.